config-examples/prometheus.yml

```diff
@@ -0,0 +1,10 @@
+global:
+  scrape_interval: 2s
+
+scrape_configs:
+  - job_name: 'bob'
+
+    scrape_interval: 2s
+
+    static_configs:
+      - targets: ['127.0.0.1:9000']
```

Comment:
Can this config be integrated into `node.yaml`?

Probably, config restructuring is required:
```
metrics:
    graphite:
        ...
    prometheus:
        ...
```

Reply:
@ikopylov it is prometheus config

@Justarone Are you sure we need this in repo? 
If yes, add comment that this is not bob config file